dev-client/src/screens/SiteNotesScreen/ReadNoteScreen.tsx

```diff
@@ -27,15 +27,18 @@ import {
 } from 'terraso-mobile-client/components/NativeBaseAdapters';
 import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
 import {ScreenScaffold} from 'terraso-mobile-client/screens/ScreenScaffold';
+import {useSelector} from 'terraso-mobile-client/store';
+import {selectProject} from 'terraso-mobile-client/store/selectors';
 
 type Props = {
-  content: string;
-  isSiteInstructions?: boolean;
+  projectId: string;
 };
 
-export const ReadNoteScreen = ({content, isSiteInstructions}: Props) => {
+export const ReadNoteScreen = ({projectId}: Props) => {
```

Comment:
Similar comment to below -- wondering if we could give this screen a more accurate name.